express-notes-api/server.ts

```diff
@@ -0,0 +1,131 @@
+import { readFile, writeFile } from 'node:fs/promises';
+import express from 'express';
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/notes', async (req, res) => {
+  try {
+    const notesArray = await read();
+    res.status(200).json(notesArray);
+  } catch (error) {
+    res.status(404).json({ error: `${error.message}` });
```

Comment:
I don't see the solution we discussed. Just not pushed yet?

Also, you should log this error since it means something went wrong that might indicate a bug.

Reply:
They're not pushed yet, they will be in the final version.